### PR TITLE
libusermode: add CLSID and IID types to printers

### PR DESCRIPTION
**Pull request by skvl**
Re-work #931

### CODE REVIEW

File: src/libusermode/printers/printers.cpp [hunk 1]
```diff
@@ -151,6 +151,35 @@ std::string UnicodePrinter::getBuffer(vmi_instance_t vmi, const access_context_t
...
+        uint32_t Data1;
+        uint16_t Data2;
+        uint16_t Data3;
+        uint8_t Data4[8];
+    } __attribute__((packed, aligned(4))) guid;
+    memset(&guid, 0, sizeof(guid));
+    vmi_read(vmi, &ctx, sizeof(guid), &guid, nullptr);
+    drakvuf_release_vmi(drakvuf);
+   const int sz = 64;
+   char stream[sz] = {0};
```
Comment (icedevml) [inline src/libusermode/printers/printers.cpp:175]: misaligned code

**Comment by icedevml** [comment]
@drakvuf-jenkins This is OK to test.

File: src/libusermode/printers/printers.cpp (hunk 1 above)
Comment (skvl) [reply src/libusermode/printers/printers.cpp]: @icedevml, fixed!

**Comment by skvl** [comment]
@icedevml, @tklengyel I have also fixed here #946 (raised because of #949)

**Pull request merged**